workhorse.make

```diff
@@ -811,7 +811,9 @@ ifeq (3.4.5,$(gcc_version))
 endif
 
 ifneq (,$(USE_SO_ATTRIBUTES))
-  LDFLAGS += -Wl,--disable-auto-import -static-libstdc++
+  ifneq (x86_64-pc-linux-gnu,$(LMI_TRIPLET))
```

Comment:
I actually think we should probably check _for_ MinGW/Cygwin platforms here rather than checking for "not Linux". I.e. compiling lmi under FreeBSD (which nobody does, but why not, in principle) still shouldn't use `--disable-auto-import`.